doc-content/enterprise-only/websphere/was-configure-embedded-engine-proc.adoc

```diff
@@ -0,0 +1,284 @@
++
+.Client dependency
+[source,xml]
+----
+<dependency>
+  <groupId>org.uberfire</groupId>
+  <artifactId>uberfire-rest-client</artifactId>
+</dependency>
+----
++
+* When creating a JAR file that includes assets, such as rules and process definitions, specify the packaging type for your Maven project as `kjar` and use `org.kie:kie-maven-plugin` to process the `kjar` packaging type located under the `<project>` element. In the following example, `${kie.version}` is the Maven library version listed in
+ifdef::PAM[]
+https://access.redhat.com/solutions/3405361[What is the mapping between Red Hat Process Automation Manager and the Maven library version?]:
+endif::[]
+ifdef::DM[]
+https://access.redhat.com/solutions/3363991[What is the mapping between Red Hat Decision Manager and the Maven library version?]:
+endif::[]
++
+[source,xml]
+----
+<packaging>kjar</packaging>
+<build>
+ <plugins>
+  <plugin>
+   <groupId>org.kie</groupId>
+   <artifactId>kie-maven-plugin</artifactId>
+   <version>${kie.version}</version>
+   <extensions>true</extensions>
+  </plugin>
+ </plugins>
+</build>
+----
+
+
+ifdef::PAM[]
+. If you use a {PROCESS_ENGINE} or {DECISION_ENGINE} with persistence support in your project, you must declare the following hibernate dependencies in the `dependencyManagement` secion of your `pom.xml` file.
++
+[id='hibernate_dependencies_with_process_engine_and_decision_engine']
+.Hibernate dependencies
+[source,xml]
+----
+<!-- hibernate dependencies -->
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-entitymanager</artifactId>
+    <version>${version.org.hibernate-4ee7}</version>
+    </dependency>
+
+    <dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-core</artifactId>
+    <version>${version.org.hibernate-4ee7}</version>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+----
+endif::[]
++
+ifdef::DM[]
+. If you use a {DECISION_ENGINE} with persistence support in your project, you must declare the following hibernate dependencies in the `dependencyManagement` secion of your `pom.xml` file.
++
+[id='hibernate_dependencies_with_decision_engine']
+.Hibernate dependencies in {DECISION_ENGINE} with persistence
+[source,xml]
+----
+<!-- hibernate dependencies -->
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-entitymanager</artifactId>
+    <version>${version.org.hibernate-4ee7}</version>
+    </dependency>
+
+    <dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-core</artifactId>
+    <version>${version.org.hibernate-4ee7}</version>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+----
+endif::[]
++
+[NOTE]
+====
+Copy the current version of the `version.org.hibernate-4ee7` property from the {PRODUCT_BA} bill of materials (BOM) Parent file to the project’s `pom.xml` file.
```

Comment:
I still want to know what the BOM parent file is and I'm pretty sure it shouldn't be capitalized. Also, don't spell out BOM here. It is defined at the top of the topic.

Reply:
@emmurphy1, You can see the BOM parent file in here: https://repository.jboss.org/nexus/service/local/repositories/product-rhdm-07_03_00/content/com/redhat/ba/bom-parent/7.3.0.GA-redhat-00002/bom-parent-7.3.0.GA-redhat-00002.pom 

In that XML file, the current version of `version.org.hibernate-4ee7` is present. So the developer can be up to date with the current version. If the user/developer want to add this version manually, he can simply copy and paste it to the project's pom.xml file.